analyzer/src/main/kotlin/managers/Conda.kt

```diff
@@ -0,0 +1,162 @@
+import com.here.ort.model.jsonMapper
+import com.here.ort.utils.CommandLineTool
+import com.here.ort.utils.ProcessCapture
+import com.here.ort.utils.log
+import com.here.ort.utils.safeDeleteRecursively
+
+import java.io.File
+import java.io.IOException
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+/**
+ * The [Conda](https://docs.conda.io/en/latest/) package manager for Python.
+ */
+class Conda(
+    name: String,
+    analyzerRoot: File,
+    analyzerConfig: AnalyzerConfiguration,
+    repoConfig: RepositoryConfiguration
+) : PackageManager(name, analyzerRoot, analyzerConfig, repoConfig), CommandLineTool {
+    class Factory : AbstractPackageManagerFactory<Conda>("Conda") {
+        override val globsForDefinitionFiles = listOf("environment.yml")
+
+        override fun create(
+            analysisRoot: File,
+            analyzerConfig: AnalyzerConfiguration,
+            repoConfig: RepositoryConfiguration
+        ) = Conda(managerName, analysisRoot, analyzerConfig, repoConfig)
+    }
+
+    // Create an environment with Conda named "ort<datetime>".
+    private val envName = run {
+        val formatter = DateTimeFormatter.ofPattern("yyyyMMddhhmmss")
+        val formattedDate = LocalDateTime.now().format(formatter)
+        "ort$formattedDate"
+    }
+
+    override fun command(workingDir: File?) = "conda"
+
+    /**
+     * Run [commandName] with arguments [commandArgs] in environment at directory [envDir].
+     */
+    fun runInEnv(envDir: File, commandName: String, vararg commandArgs: String): ProcessCapture =
+        ProcessCapture(command(), "run", "--name", envDir.name, commandName, *commandArgs).requireSuccess()
+
+    override fun resolveDependencies(definitionFile: File): ProjectAnalyzerResult? {
+        val workingDir = definitionFile.parentFile
+        val envDir = setupEnv(definitionFile)
+
+        val support = object : PythonSupport(managerName, analysisRoot, command(workingDir), envDir) {
+            override fun runInEnv(workingDir: File, commandName: String, vararg commandArgs: String): ProcessCapture =
+                this@Conda.runInEnv(envDir, commandName, *commandArgs)
+        }
+
+        val result = support.resolveDependencies(definitionFile)
+
+        // Remove the env by simply deleting the directory.
+        envDir.safeDeleteRecursively()
+
+        return result
+    }
+
+    private fun setupEnv(definitionFile: File): File {
+        // Create an out-of-tree environment.
+        log.info { "Creating a Conda env for $definitionFile..." }
+        val envDir = createEnv(definitionFile)
+        val install = installDependencies(definitionFile, envDir)
+
+        if (install.isError) {
+            log.debug {
+                // Pip writes the real error message to stdout instead of stderr.
+                "First try to install dependencies using Conda failed with:\n${install.stdout}"
+            }
+
+            if (install.isError) {
```

Comment:
If the condition in line 102 gets evaluated does it always return `true`? ... as the condition in 96 also must have been `true`.